stor/dx.py

```diff
@@ -1006,6 +1062,9 @@ def walkfiles(self,
         Returns:
              Iter[DXPath]: Iterates over listed files that match an optional pattern.
         """
+        if not self.project:  # for empty paths
+            raise ValueError('list, walkfiles and glob are not supported on empty paths. '
```

Comment:
why not allow `list()` on root?

Reply:
Because it seems like overkill. We introduced empty path to just be able to see projects a user has access to. list() would involve invoking list on each of those projects (which is ~50 times for some users). List might take too long to return even. Can do `stor ls dx:// | xargs -L1 stor list` if really need to